lnbits/db.py

```diff
@@ -83,33 +83,41 @@ def datetime_to_timestamp(self, date: datetime.datetime):
             return time.mktime(date.timetuple())
         return "<nothing>"
 
+    @property
+    def is_postgres(self) -> bool:
```

Comment:
this information should never be leaked outside this file

Reply:
also the function is not doing what it is announcing, cockroach is not postgres, NACK